### DISCUSSION TITLE
Can I disable win+down window minimization ?

### DISCUSSION THREAD

**Discussion by ghost** [AvaloniaUI/Avalonia#5995] [Q&A]
I set CanResize="False" but it didnt help.

**Comment by blackuGT** [comment]
Do you implemented `  CanResize="False" ` inside `<Window>` ?

**Reply by ghost** [reply]
Yes

**Reply by blackuGT** [reply]
Show me code inside Window.

**Reply by ghost** [reply]
![изображение](https://user-images.githubusercontent.com/50862104/121536255-92b94b00-ca0b-11eb-9558-aa3382413e65.png)

**Reply by blackuGT** [reply]
Weird. I added code like yours and `CanResize` property works really wel if ofc `HasSystemDecorations` is removed.

**Reply by ghost** [reply]
My window become like this
![изображение](https://user-images.githubusercontent.com/50862104/121539887-b4680180-ca0e-11eb-8a33-3c52ea9e193f.png), when I run code above and click win+down.

If I remove `HasSystemDecorations` then I get the same.
If I remove `HasSystemDecorations` and `SystemDecorations` then I get 
![изображение](https://user-images.githubusercontent.com/50862104/121540459-2b04ff00-ca0f-11eb-9c9d-7351bcea90da.png)

**Comment by maxkatz6** [comment]
If it's possible with Win32 API, you can do it by yourself. 
You can get window handle with Window.PlatformImpl.Handle.Handle.
